.github/workflows/docker-release.yaml

```diff
@@ -0,0 +1,46 @@
+name: docker_release
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore:
+      - 'README.md'
+      - 'CHANGELOG.md'
+    tags: [ 'v*.*.*' ]
```

Comment:
This creates a tag for each release. The docker metadata action will also set the latest tag to the latest release.